modules/platform/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/extension/CSVZipImporter.java

```diff
@@ -241,9 +254,16 @@ protected Serializable getFieldValue(Field field, String stringValue, ZipFile zi
                 log.warn(String.format("Unsupported field type '%s'", type));
                 return null;
             }
-        } else if (type.isComplexType() && TypeConstants.CONTENT.equals(field.getName().getLocalName())) {
+        } else if (type.isComplexType()) {
+            boolean returnEarly = true;
+            if (TypeConstants.CONTENT.equals(field.getName().getLocalName())) {
+                returnEarly = false;
+            }
             ZipEntry blobIndex = zip.getEntry(stringValue);
-            if (blobIndex != null) {
+            if (!returnEarly) {
+                returnEarly = blobIndex == null;
+            }
```

Comment:
You can just replace these 3 lines with `returnEarly |= blobIndex == null`

Reply:
Nice :) Updated